apps/src/gamelab/spritelab/commands.js

```diff
@@ -0,0 +1,9 @@
+import {commands as spriteCommands} from './spriteCommands';
+
+let commands = module.exports;
+
+commands.setBackground = function(color) {
+  this.background(color);
+};
+
+Object.assign(commands, spriteCommands);
```

Comment:
Probably each category in the toolbox will have a commands file, and then they'll all just be conglomerated together here

Reply:
So will commands.setBackground go in a different file?

Reply:
yeah, sorry I haven't moved that over yet. that will eventually be moved to worldCommands.js